native-sql-engine/core/src/test/scala/org/apache/spark/sql/test/SharedSparkSession.scala

```diff
@@ -94,6 +96,7 @@ trait SharedSparkSessionBase
     conf.set(
       StaticSQLConf.WAREHOUSE_PATH,
       conf.get(StaticSQLConf.WAREHOUSE_PATH) + "/" + getClass.getCanonicalName)
+    conf.set(GazellePluginConfig.getSessionConf.enableUDFKey, "true")
```

Comment:
It looks the config is cached for some reason. So we set the config here to make sure this config is turned on, as required by columnar udf test.